tools/api-markdown-documenter/src/api-item-transforms/helpers/Helpers.ts

```diff
@@ -93,15 +96,15 @@ export function createSignatureSection(
 	if (apiItem instanceof ApiDeclaredItem) {
 		const signatureExcerpt = apiItem.getExcerptWithModifiers();
 		if (signatureExcerpt !== "") {
-			const contents: DocumentationNode[] = [];
+			const contents: SectionContent[] = [];
 
 			contents.push(
 				FencedCodeBlockNode.createFromPlainText(signatureExcerpt.trim(), "typescript"),
 			);
 
 			const renderedHeritageTypes = createHeritageTypesParagraph(apiItem, config);
```

Comment:
Should createHeritageTypesParagraph be renamed now that its return type changed? (or maybe inline it?)